python/taichi/main.py

```diff
@@ -403,11 +403,8 @@ def format(self, arguments: list = sys.argv[2:]):
         """Reformat modified source files"""
         parser = argparse.ArgumentParser(description=f"{self.format.__doc__}")
         parser.add_argument(
-            '-d',
-            '--diff',
-            required=False,
-            default=None,
-            dest='diff',
+            'diff',
+            nargs='?',
```

Comment:
@rexwangcc Format server uses `ti format <diff>`, you changed it into `ti format -d <diff>`, so I fix this.